.github/actions/annotate/action.yml

```diff
@@ -1,5 +1,5 @@
 name: 'Annotate Errors'
-description: 'Annotates errors if an errrors.txt file is present'
+description: 'Annotates errors if an errors.txt file is present'
```

Comment:
Sometimes projects try to be cute. Often even if a project tries to be cute, it still contains typos. This appears to be the latter.